app/views/how-tos/index.html

```diff
@@ -40,7 +40,10 @@ <h2 class="nhsuk-heading-s">Download and install</h2>
           <a href="/install/advanced">Advanced install guide</a>
         </li>
         <li>
-          <a href="/install/nhs-england-windows-laptops">Set up the prototype kit on an NHS England Windows laptop</a>
+          <a href="/install/nhs-england-windows-laptops">Set up the kit on an NHS England Windows laptop</a>
+        </li>
+        <li>
+          <a href="/install/codespaces/index">Install and use the kit with your web browser</a>
```

Comment:
This doesn't need `/index` on the end.

```suggestion
          <a href="/install/codespaces">Install and use the kit with your web browser</a>
```